### ISSUE TITLE
The highlights grid  doesn't appear over  the scrollbar but on the left side of the window

### ISSUE THREAD

**Issue by mzso** [Quicksaver/FindBar-Tweak#293]
Hi!

This is fairly recent issue on moz-central. So now that development halted is there anyone out there who knows what's going on?
It might be a simple fix since the feature itself is not broken, but appears to not find the scrollbar.

**Comment by pintassilgo** [comment]
Mozilla changed the LocaleService API, so outdated FindBar Tweak code can't detect the user language, reverting to (code) default RTL (right-to-left), that is, thinking that scrollbar is in the left side.

https://hg.mozilla.org/mozilla-central/diff/20075fb33fc7/toolkit/modules/Services.jsm

https://bugzilla.mozilla.org/show_bug.cgi?id=1337551

To fix this, change **line 763** of **resource/modules/content/grid.jsm**
from
`var appLocale = Services.locale.getApplicationLocale().getCategory("NSILOCALE_MESSAGES");`
to
`var appLocale = Services.locale.getAppLocale();`

Or simply set about:config
`layout.scrollbar.side = 2`

**Comment by mzso** [comment]
[**@pintassilgo**](https://github.com/pintassilgo) commented on [Mar 25, 2017, 9:23 AM GMT+1](https://github.com/Quicksaver/FindBar-Tweak/issues/293#issuecomment-289197642 "2017-03-25T08:23:33Z - Replied by Github Reply Comments"):
> Mozilla changed the LocaleService API, so outdated FindBar Tweak code can't detect the user language, reverting to (code) default RTL (right-to-left), that is, thinking that scrollbar is in the left side.
> 
> [https://hg.mozilla.org/mozilla-central/diff/20075fb33fc7/toolkit/modules/Services.jsm](https://hg.mozilla.org/mozilla-central/diff/20075fb33fc7/toolkit/modules/Services.jsm)
> 
> [https://bugzilla.mozilla.org/show_bug.cgi?id=1337551](https://bugzilla.mozilla.org/show_bug.cgi?id=1337551)
> 
> To fix this, change **line 763** of **resource/modules/content/grid.jsm**  
> 
> from  
> 
> `var appLocale = Services.locale.getApplicationLocale().getCategory("NSILOCALE_MESSAGES");`  
> 
> to  
> 
> `var appLocale = Services.locale.getAppLocale();`
> 
> Or simply set about:config  
> 
> `layout.scrollbar.side = 2`

Thanks!
Sadly, this change in the code doesn't seem to help. The highlights still appear on the left.

**Comment by mzso** [comment]
So, since this change doesn't work can I expect negative effects in some circumstances from setting using "layout.scrollbar.side = 2"?

**Comment by harl** [comment]
http://kb.mozillazine.org/Layout.scrollbar.side

Basically, if LTR is the only content direction you care about, then it's rather unlikely you will see any negative effects from changing that variable. - On the other hand, if it behaves differently from your expectations on RTL pages (or whatever), then it's pretty easy to change it back to 0 or 1 (or 3 if 0 or 1 don't work).

You can set it to 1 and see if that solves this issue without interfering with any content, as it should have the same effect as setting it to 2 on LTR content but still let the scrollbar change accordingly when needed.

Bottom line, set it to 1 and it might just work out for you, or set it to 2 to force it to the right side.

**Comment by mzso** [comment]
[**@harl**](https://github.com/harl) commented on [Apr 24, 2017, 1:54 PM GMT+2](https://github.com/Quicksaver/FindBar-Tweak/issues/293#issuecomment-296636554 "2017-04-24T11:54:28Z - Replied by Github Reply Comments"):
> [http://kb.mozillazine.org/Layout.scrollbar.side](http://kb.mozillazine.org/Layout.scrollbar.side)
> 
> Basically, if LTR is the only content direction you care about, then it's rather unlikely you will see any negative effects from changing that variable. - On the other hand, if it behaves differently from your expectations on RTL pages (or whatever), then it's pretty easy to change it back to 0 or 1 (or 3 if 0 or 1 don't work).
> 
> You can set it to 1 and see if that solves this issue without interfering with any content, as it should have the same effect as setting it to 2 on LTR content but still let the scrollbar change accordingly when needed.
> 
> Bottom line, set it to 1 and it might just work out for you, or set it to 2 to force it to the right side.

Thanks. layout.scrollbar.side;1 seems to work fine
